app/lib/draggable.cjsx

```diff
@@ -7,7 +7,6 @@ module.exports = React.createClass
   _previousEventCoords: null
 
   propTypes:
-    children: React.PropTypes.component.isRequired
```

Comment:
Yeah looks like it's no longer in the valid prop-types https://facebook.github.io/react/docs/reusable-components.html